src/java.base/share/classes/java/lang/Runtime.java

```diff
@@ -90,9 +90,11 @@
  *
  * <p>When the JVM terminates, all threads are immediately prevented from executing any further
  * Java code. This includes shutdown hooks as well as daemon and non-daemon threads. The
- * threads' current methods do not complete normally or abruptly; no {@code finally} clause
- * of any method is executed, nor is any {@linkplain Thread.UncaughtExceptionHandler
- * uncaught exception handler}.
+ * threads' current methods do not complete normally or abruptly.
+ * {@linkplain Thread.UncaughtExceptionHandler Uncaught exception handlers} are not run,
+ * no {@code finally} clause of any method is executed, and try-with-resources
+ * blocks do not {@linkplain AutoCloseable close} their resources.
+ * Other unexpected behaviors may also result.
```

Comment:
No sorry that doesn't work for me. It sounds like you might get unexpected things happening, when in fact it is things not happening that might surprise someone (who doesn't understand what "immediately prevented from executing any further Java code" means). The list of examples is getting too long to handle in sentence structure so I suggest an actual list (not sure of list syntax). Suggestion:

```
 * Java code. This includes shutdown hooks as well as daemon and non-daemon threads. This 
 * means, for example, that:
 * - threads' current methods do not complete normally or abruptly
 *  - no {@code finally} clause of any method is executed
 *  - no {@linkplain Thread.UncaughtExceptionHandler uncaught exception handler} is executed
 *  - no try-with-resources blocks {@linkplain AutoCloseable close} their resources
```

Reply:
I think it's reasonable to position these as examples and to put them into list form.

(wordsmithing) For parallel structure I'd suggest "X does not occur" instead of "no X occurs" in all but the first item, specifically (markup elided):

 - threads' current methods do not complete normally or abruptly;
 - finally clauses are not executed;
 - uncaught exception handlers are not executed; and
 - resources opened by try-with-resources statements are not closed.